mqtt/mqtt.js

```diff
@@ -101,29 +107,21 @@ nats.subscribe('channel.>', {'queue':'mqtts'}, function (msg) {
 aedes.authorizePublish = function (client, packet, publish) {
     // Topics are in the form `channels/<channel_id>/messages`
     // Subtopic's are in the form `channels/<channel_id>/messages/<subtopic>`
-    var channel = /^channels\/(.+?)\/messages\/?.*$/.exec(packet.topic);
+    var channel = parseMqttTopic(packet.topic)
```

Comment:
This name is confusing. Regexp does not return only channel ID in the selection, but subtopic as well from what I see.

Something like `topicParts` would make more sense.

Reply:
RegExp return and array with channel name and the topic parts, maybe i can raneme it channelParts, what do you thing?

Reply:
Yes, this is good also. Just not `channel`, as it is missleadning.